packages/vulnerabilities/preProcess.js

```diff
@@ -0,0 +1,20 @@
+var request = require("request");
+const fs = require("fs");
+const SPEC = "apiSpec.json";
+const PRODSPEC = "https://cloud.redhat.com/api/vulnerability/v1/openapi.json";
```

Comment:
Do you think you could pass the URL to this file as argument?

Reply:
Sure, it's been updated